email_api/email.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const router = express.Router();
+nodeMailer = require('nodemailer');
+var sendemail = require('./send_email');
```

Comment:
var sendemail should be const type since we are not going to modify anything in this object.
Follow camel case convention as well

Reply:
var sendemail is now changed